repo-scripts/size-analysis/analysis.ts

```diff
@@ -66,10 +70,16 @@ const argv = yargs
       alias: 'ib',
       desc: 'support for adhoc analysis. requires a path to a bundle file'
     },
+    ci: {
+      type: 'boolean',
+      alias: 'ci',
+      default: false,
+      desc:
+        "when enabled, the binary size report is not persisted on file system; Instead, it's uploaded to CI backend"
+    },
```

Comment:
Uploading to backend should be handled by a different script, not the analyzer.
We should make it possible to use the size-analysis tool programmatically by allowing another script to import it, execute a function and get a result object back.